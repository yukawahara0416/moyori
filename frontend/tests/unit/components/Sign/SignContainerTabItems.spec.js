import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vuex from 'vuex'
import Component from '@/components/Sign/SignContainerTabItems.vue'

const localVue = createLocalVue()
localVue.use(Vuex)

let wrapper

beforeEach(() => {
  wrapper = shallowMount(Component, {
    localVue
  })
})

// describe('computed', () => {
//   it('childTabs', () => {})
// })

// describe('emit', () => {
//   it('childTabs', () => {})
// })

describe('template', () => {
  it('snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
