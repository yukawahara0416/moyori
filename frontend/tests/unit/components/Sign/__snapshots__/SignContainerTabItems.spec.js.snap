// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template snapshot 1`] = `
<v-tabs-items-stub
  activeclass="v-window-item--active"
  nexticon="$next"
  previcon="$prev"
  value="tab-1"
>
  <v-tab-item-stub
    value="tab-1"
  >
    <sign-container-tab-items-sign-in-stub />
  </v-tab-item-stub>
   
  <v-tab-item-stub
    value="tab-2"
  >
    <sign-container-tab-items-sign-up-stub />
  </v-tab-item-stub>
</v-tabs-items-stub>
`;
